tests/io/test_readwrite.py

```diff
@@ -275,45 +276,6 @@ def test_overwrite_onto_non_zarr_file(self, full_sdata):
             with pytest.raises(ValueError):
                 full_sdata.write(f1)
 
-    def test_incremental_io_with_backed_elements(self, full_sdata):
-        # addressing https://github.com/scverse/spatialdata/issues/137
-        # we test also the non-backed case so that if we switch to the
-        # backed version in the future we already have the tests
-
-        with tempfile.TemporaryDirectory() as tmpdir:
-            f = os.path.join(tmpdir, "data.zarr")
-            full_sdata.write(f)
-
-            e = full_sdata.images.values().__iter__().__next__()
-            full_sdata.add_image("new_images", e, overwrite=True)
-            # support for overwriting backed images has been temporarily removed
-            with pytest.raises(ValueError):
-                full_sdata.add_image("new_images", full_sdata.images["new_images"], overwrite=True)
-
-            e = full_sdata.labels.values().__iter__().__next__()
-            full_sdata.add_labels("new_labels", e, overwrite=True)
-            # support for overwriting backed labels has been temporarily removed
-            with pytest.raises(ValueError):
-                full_sdata.add_labels("new_labels", full_sdata.labels["new_labels"], overwrite=True)
-
-            e = full_sdata.points.values().__iter__().__next__()
-            full_sdata.add_points("new_points", e, overwrite=True)
-            # support for overwriting backed points has been temporarily removed
-            with pytest.raises(ValueError):
-                full_sdata.add_points("new_points", full_sdata.points["new_points"], overwrite=True)
-
-            e = full_sdata.shapes.values().__iter__().__next__()
-            full_sdata.add_shapes("new_shapes", e, overwrite=True)
-            full_sdata.add_shapes("new_shapes", full_sdata.shapes["new_shapes"], overwrite=True)
-
-            # commenting out as it is failing
-            # f2 = os.path.join(tmpdir, "data2.zarr")
-            # sdata2 = SpatialData(table=full_sdata.table.copy())
-            # sdata2.write(f2)
-            # del full_sdata.table
-            # full_sdata.table = sdata2.table
-            # full_sdata.write(f2, overwrite=True)
-
```

Comment:
Let's remember that this code was removed here because we will need to add similar tests when working with incremental IO or with lazy copy.